PWGCF/FemtoUniverse/TableProducer/femtoUniverseProducerTask.cxx

```diff
@@ -1664,6 +1664,14 @@ struct femtoUniverseProducerTask {
   }
   PROCESS_SWITCH(femtoUniverseProducerTask, processTrackMCTruth, "Provide MC data for MC truth track analysis", false);
 
+  void processTrackMCGen(aod::McCollision const& col,
```

Comment:
Hi, I am doing PR now #7534. I would like to accept this one, but it would overwrite mine. Could you add those changes to yours, please?

Reply:
We can not add your changes in two different PRs, it will certainly result in conflicts.